runtime/src/consensus/state/staking.rs

```diff
@@ -0,0 +1,72 @@
+use std::collections::BTreeMap;
+
+use anyhow::{anyhow, Result};
```

Comment:
Can we use `thiserror` instead and define the errors?

Reply:
ok

Reply:
fixed